DigitalLearningSolutions.Data/Services/SearchSortFilterPaginateService.cs

```diff
@@ -16,17 +18,27 @@ SearchSortFilterAndPaginateOptions searchSortFilterAndPaginateOptions
 
     public class SearchSortFilterPaginateService : ISearchSortFilterPaginateService
     {
+        private readonly IConfiguration configuration;
+
+        public SearchSortFilterPaginateService(IConfiguration configuration)
+        {
+            this.configuration = configuration;
+        }
+
         public SearchSortFilterPaginationResult<T> SearchFilterSortAndPaginate<T>(
             IEnumerable<T> items,
             SearchSortFilterAndPaginateOptions searchSortFilterAndPaginateOptions
         ) where T : BaseSearchableItem
         {
-            var itemsToReturn = items;
+            var allItems = items.ToList();
+            var itemsToReturn = allItems;
             string? appliedFilterString = null;
+            var javascriptShouldBeEnabled =

```

Comment:
Minor, but as there are some pages that will still use JS for non-search/sort/filter/paginate functionality, this could maybe be renamed to be more specific.

Reply:
I don't think it matters a huge amount since it's a variable within the SearchSortFilterAndPaginate method, but updated it anyway.